misc/what_is_rdf_and_why.md

```diff
@@ -0,0 +1,300 @@
+<https://example.org/people/adaLovelace>   <https://schema.org/knowsAbout>                     "programmering"@nb .
+```
+
+**Forkortert i Turtle**
+
+```turtle
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix foaf:   <http://xmlns.com/foaf/0.1/> .
+@prefix xsd:    <http://www.w3.org/2001/XMLSchema#> .
+@prefix schema: <https://schema.org/> .
+
+@prefix people: <https://example.org/people/> .
+
+people:adaLovelace rdf:type foaf:Person ;
+    foaf:name         "Ada Lovelace" ;
+    schema:birthDate  "1815-12-10"^^xsd:date ;
+    schema:knows      people:charlesBabbage ;
+    schema:knowsAbout "musikk"@nb , "matematikk"@nb , "programmering"@nb .
+```
+
+Men husk at disse to utgavene representerer akkurat den samme informasjonen, og at det hele veien er URI-er, verdier og tripler i bånn.
+
+### Blanke noder (ressurser uten URI)
+
+I tillegg er det greit å vite om såkalt blanke noder. Dette er ressurser vi ønsker å beskrive, men som vi ikke ønsker å gi en global og unik identifikator.
+
+I Ada Lovelace-eksemplet ønsker vi å oppgi navnene til Lovelace sine barn, Byron, Anna Isabella og Ralph Gordon, men vi vil ikke å lage nye URI-er for dem.
+Da kan vi lage blanke noder ved hjlpe av hakeparenteser `[` og `]`, og oppgi navnene innenfor der.
+
+```turtle
+@prefix rdf:    <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix foaf:   <http://xmlns.com/foaf/0.1/> .
+@prefix schema: <https://schema.org/> .
+
+@prefix people: <https://example.org/people/> .
+
+people:adaLovelace rdf:type foaf:Person ;
+    # ...kommentert vekk
+    schema:children [
+      a foaf:Person ;
+      foaf:name "Byron" ;
+    ] ,
+    [
+      a foaf:Person ;
+      foaf:name "Anna Isabella" ;
+    ] ,
+    [
+      a foaf:Person ;
+      foaf:name "Ralph Gordon" ;
+    ] .
+```
+
+> **Tips:** `a` er en forkortelse for `rdf:type` i Turtle.
+
+## RDF-serialiseringer/syntakser
+
+Det fins flere ulike måter å uttrykke RDF-grafer. I tabellen under har vi listet noen av de vanligste syntaksene for å serialisere RDF.
+
+RDF-verktøy og biblioteker kan lese til og skrive fra en eller flere av disse syntaksene. De facto standard er RDF/XML, men Turtle støttes også av de aller fleste verktøy.
+
+| Navn      | Mediatype               | File extension |
+| --------- | ----------------------- | -------------- |
+| Turtle    | `text/turtle`           | `.ttl`         |
+| RDF/XML   | `application/rdf+xml`   | `.rdf`         |
+| JSON-LD   | `application/ld+json`   | `.jsonld`      |
+| N-Triples | `application/n-triples` | `.nt`          |
+| N-Quads   | `application/n-quads`   | `.nq`          |
+| TriG      | `application/trig`      | `.trig`        |
+| Notation3 | `text/n3;charset=utf-8` | `.n3`          |
+
+## Eksterne ressurser
+
+Deler av denne teksten er basert på W3C sin [RDF 1.1 Primer](https://www.w3.org/TR/rdf11-primer/), en ypperlig ressurs som er verdt å lese.
```

Comment:
Bare en ide: Kunne det gitt mening å skrive noe om "Vanlige feil/hjelp til feilsøking" ? Evt på siden for "hvordan skrive din egen beskrivelse"

Reply:
Tenker da særlig på . og ; 😅 Også kunne man kanskje igjen nevnt validatoren som et verktøy for feilsøking